src/Core/Framework/Script/Api/StoreApiCacheKeyHook.php

```diff
@@ -30,7 +30,7 @@ class StoreApiCacheKeyHook extends OptionalFunctionHook implements SalesChannelC
 
     private string $name;
 
-    private ?string $cacheKey;
+    private ?string $cacheKey = '';
```

Comment:
why not:

```suggestion
    private ?string $cacheKey = null;
```

if it is nullable, the initial value should be `null`, shouldn't it?

Reply:
Maybe another solution could also be in the getter, using something like:
`return $this->cacheKey ?? null;` (untested should work), or otherwise:
`return (isset($this->cacheKey)) ? $this->cacheKey : null;`

But yes, I think it should at least be the value `null` and not an empty string.